doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
-solve. For example, many companies are actively developing specialized 
-:abbr:`Application-Specific Integrated Circuits (ASICs)` to speed-up 
-training (one kind of ASIC) or to reduce inference latency (another kind 
-of ASIC) in their cloud-based or local data centers. This approach works 
-great for :abbr:`Cloud Service Providers (CSPs)` and others that have 
-considerable budgets to invest in researching and building new hardware; 
-however, it creates a significant burden on the developer who needs to 
-invest in adapting the context of their model for training and then for 
-inference, to figure out at least two data-cycle pipelines or deployment 
-scenarios, and to decide what trade-offs to make when and where.  
-
-The second approach to making deep learning more efficient is to design a  
-software stack that lets the :abbr:`Neural Network (NN)` adapt to whatever 
-compute resources are available and deliver performance via software 
-optimization. The nGraph Compiler stack is our solution to this second 
-approach: it provides an inherently efficient graph-based compilation 
-infrastructure designed to be compatible with many upcoming DL ASICs while 
-also unlocking a massive performance boost on any existing hardware targets 
-in a network, whether they are CPUs, GPUs, or other custom silicon. nGraph 
-provides optimization opportunities at the graph level, where the 
-network-to-device compilation can be managed with a series of "subgraphs"
-that can be handled in either a static or a dynamic manner. With our 
-:doc:`../ops/index` and graph-based infrastructure for neural networks, 
-it's also possible to extract context semantics that make it much easier to 
-work with many of the new and emerging problems in Deep Learning including 
-larger datasets, data that must be encrypted, and data-in-motion. Our solution 
-also addresses the scalability issue with kernel libraries, the current 
-popular solution to accelerating deep learning performance. 
+Currently, there are two standard approaches for developers to accelerate deep learning performance:
 
+**Design hardware solutions dedicated to deep learning computation**
+
+Established chip manufacturers such as Intel and hardware startups are actively developing Application Specific Integrate Circuits (ASICs) to accelerate the performance of deep learning for both training and inference.
+
+**Optimize software to accelerate performance**
+
+nGraph, an open-source graph compiler, is our solution to deliver performance via software optimization. nGraph was created to provide developers with a way to accelerate the performance of upcoming, deep learning ASICs via software and to provide a significant increase in performance for standard hardware targets such as CPUs and GPUs. For deploying scalable AI solutions, nGraph uses kernel libraries which are currently a popular method to improve deep learning performance. 
```

Comment:
Begin with "ngraph, an open source (no hypen) *deep learning* graph compiler"

Reply:
Minor nitpick:  I don't like the comma after "upcoming"

Reply:
@aprocter the reference to PlaidML is appropriate and welcome here.